Automattic/jetpack
# VideoPress: add loading state to connect button righ after user clicks on it

Fixes https://github.com/Automattic/jetpack/issues/26949

#### Changes proposed in this Pull Request:

* 
VideoPress: add loading state to connect button right after user clicks on it

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Ensure your site is disconnected (easily doable from My Jetpack)

<img width="738" alt="Screen Shot 2022-10-20 at 07 47 31" src="https://user-images.githubusercontent.com/77539/196928710-15f20395-95a3-4921-9766-63fa80c1ded3.png">

* Go to VideoPress dashboard
* You should see the pricing section 
* Click on the `Start for free` button
* Confirm the loading state shows right after clicking on it

https://user-images.githubusercontent.com/77539/196927895-bc2e42db-e988-428a-99f3-8606a6b87a5a.mov
